arelle/ValidateXbrl.py

```diff
@@ -53,14 +53,14 @@ class ValidateXbrl:
 
     authority: str
     authParam: dict[str, Any]
-    domainMembers: list
+    domainMembers: set = set()  # concepts which are dimension domain members
```

Comment:
My previous comment applies to this, as well. Initializing here turns this into a class field that's no longer scoped to a instance.
```python
    instance_1 = ValidateXbrl(Mock())
    instance_1.domainMembers.add('domain_member_1')

    instance_2 = ValidateXbrl(Mock())
    instance_2.domainMembers.add('domain_member_2')

    print(instance_1.domainMembers)
    # {'domain_member_1', 'domain_member_2'}
    print(instance_2.domainMembers)
    # {'domain_member_1', 'domain_member_2'}
```

Reply:
Thanks for the clarification, I didn't know this.